pkgs/tools/typesetting/tex/texlive/tl2nix.sed

```diff
@@ -36,6 +36,35 @@ $a}
   /^catalogue-version/s/[\#,:\(\)]//g
   s/^catalogue-version_(.*)/  version = "\1";/p
 
+  /^catalogue-license/{
+        # wrap licenses in quotes
+        s/ ([^ ]+)/ "\1"/g
+        # adjust naming as in nixpkgs, the full text of the licenses are available at https://www.ctan.org/license/${licenseName}
+        s/"(cc-by(-sa)?-[1-4])"/"\10"/g
+        s/"apache2"/"asl20"/g
+        s/"artistic"/"artistic1"/g
+        s/"bsd"/"bsd3"/g          # license text does not match exactly, but is pretty close
+        s/"bsd4"/"bsdOriginal"/g
+        s/"collection"//g         # used for the empty collection metapackages
```

Comment:
Actually collection refers to packages made up of disparate sources with different licenses (e.g. https://ctan.org/pkg/eijkhout). The collection meta packages do not have licenses attached. I am not sure, but maybe we can use `"free"` here. We can assume that if the package is on TeX Live, then is is at least `"free"` – can we?

Reply:
Oh, I just looked at the license description and assumed that it was used for the collections in texlive. Good catch!
TeXLive's [LICENSE.TL](https://tug.org/texlive/LICENSE.TL) file says that to their knowledge, everything in texlive is free software. I'll go ahead and use "free" for this then